packages/blueprints/src/showstyle0/migrations/util.ts

```diff
@@ -144,8 +144,8 @@ export function createAdLibHotkey(
 	return {
 		_id: makeActionTriggerId('adLib', sourceLayerIds.join('_'), !!globalAdLib, pick),
 		_rank: rankCounter++ * 1000,
-		actions: [
-			{
+		actions: {
+			[PlayoutActions.adlib]: {
```

Comment:
I think this will _technically_ work, but may be confusing to the reader? I think this should preferably be some sort of a stable ID.

Reply:
I agree with this, it's a bit of an indirection. I would favor having an enum with the various action id's but even just a hardcoded string here would be more understandable to me.